README.md

```diff
@@ -5,16 +5,12 @@ It's a slightly modified version of [Making your own ipfs service](https://ipfs.
 ### Install prerequisites
 
 * [Install IPFS from source](https://github.com/ipfs/go-ipfs/#download--compile-ipfs)
-* Install repo migration tool
-```
-go get -u github.com/ipfs/fs-repo-migrations
-```
 
 ### Initialize IPFS repo for both server and client
 ```
-IPFS_PATH=.ipfs-repo-server ipfs init && fs-repo-migrations
-IPFS_PATH=.ipfs-repo-client ipfs init && fs-repo-migrations
-sed -i .prev "s/4001/4002/" .ipfs-repo-client/config # client should listen on different port than server
+IPFS_PATH=.ipfs-repo-server ipfs init
+IPFS_PATH=.ipfs-repo-client ipfs init
+sed -i "s/4001/4002/" .ipfs-repo-client/config # client should listen on different port than server
```

Comment:
It gives me `sed: 1: ".ipfs-repo-client/config": invalid command code .` on Mac.